app/components/page-footer/template.hbs

```diff
@@ -9,6 +9,11 @@
   <a role="button" class="btn btn-sm btn-link" target="blank" href="{{forumBase}}">{{t 'pageFooter.forums'}}</a>
   <a role="button" class="btn btn-sm btn-link" target="blank" href="{{slackBase}}">{{t 'pageFooter.slack'}}</a>
 {{/if}}
+
+{{#if showWechat}}
```

Comment:
This needs to be inside the `{{#if settings.isRancher}}` block, so it's not shown for private-labels (since it has our name in the icon, and points to our community..)